Source/WebCore/page/FocusController.cpp

```diff
@@ -1113,13 +1113,13 @@ void FocusController::findFocusCandidateInContainer(Node& container, const Layou
     current.visibleNode = focusedNode;
 
     unsigned candidateCount = 0;
-    for (; element; element = (element->isFrameOwnerElement() || canScrollInDirection(element, direction))
+    for (; element; element = (is<HTMLFrameOwnerElement>(element) || canScrollInDirection(element, direction))
```

Comment:
It reports a style error for this line, but that is not code I'm technically changing. Happy to take suggestions though.

Reply:
Should be `is<HTMLFrameOwnerElement>(*element)`. You added an unnecessary null check.